samples/pom.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>readyapi4j</artifactId>
+        <groupId>com.smartbear.readyapi</groupId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>readyapi4j-samples</artifactId>
+    <packaging>pom</packaging>
+    <modules>
+        <module>java</module>
+        <module>groovy</module>
+        <module>cucumber</module>
+        <module>test-server-maven-plugin</module>
```

Comment:
I agree we should rename it - I think our maven modules almost always have too long names - but why would the test server plugin be called readyapi4j-maven-plugin and this one just maven-plugin?